PeerioTechnologies/peerio-mobile
# fix(medcryptor signup): restart signup if user enters duplicate AHRPA

#### Relevant info and issue/PR links 
 https://github.com/PeerioTechnologies/peerio-icebear/pull/214
#### Testing instructions  
Try to sign up 2 users with same AHRPA. Second time should show duplicate AHRPA error and restart signup process.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
